core/tasks.py

```diff
@@ -207,7 +208,13 @@ def _score_submission(submission):
     return submission
 
 
-@shared_task(soft_time_limit=60 * 20, time_limit=60 * 21)
+@shared_task(
+    soft_time_limit=60 * 20,
+    time_limit=60 * 21,
+    autoretry_for=(SMTPServerDisconnected,),
+    default_retry_delay=10,
```

Comment:
[The docs](https://docs.celeryproject.org/en/stable/userguide/tasks.html#automatic-retry-for-known-exceptions) seem to suggest that we use `retry_backoff=True` (new in Celery 4.4) instead.

I think that would give us retry times of `[1, 2, 4, 8, 16]`, instead of always `10`.

I don't feel strongly, what do you think?